### ISSUE TITLE
Syntax Error - Indicate Location

### ISSUE THREAD

**Issue by bgribaudo** [microsoft/vscode-powerquery-sdk#286]
### Preflight Checklist

- [X] I have installed the [latest version of Power Query SDK](https://github.com/microsoft/vscode-powerquery-sdk/releases/latest).
- [X] I have checked existing resources, including the [common issues](https://learn.microsoft.com/en-us/power-query/commonissues) and the [release notes](https://github.com/microsoft/vscode-powerquery-sdk/releases).
- [X] I have searched for [similar issues](https://github.com/microsoft/vscode-powerquery-sdk/issues).

### Problem

When a .pq file has a syntax error, the error that is displayed may not identify which file in the project has the problem.

## Example
Notice how below reports a syntax error, but not where it is to be found:
````
[5:58:24 PM]	[Info]	[Task found] c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.2.3-win32-x64\.nuget\Microsoft.PowerQuery.SdkTools.2.114.4\tools\PQTest.exe run-test
				--extension "Some.mez"
				--queryFile "Some.query.pq"
				--prettyPrint
[5:58:26 PM]	[Error]	[Task exited abnormally] pqtest run-test pid(23484) exit(3762504530) stderr: 
Unhandled Exception: System.InvalidOperationException: Unable to parse the document
   at PQTest.Common.Utilities.Parse(String mashupDocument)
   at PQTest.Common.Utilities.GetDocumentKind(String code)
   at PQTest.Common.QueryFileAnalyzer.PerformAnalysis()
   at PQTest.Common.Commands.PrepareTests(Configuration configuration)
   at PQTest.Common.Commands.<>c__DisplayClass14_0.<RunTestBattery>b__0()
   at PQTest.Common.Commands.ConfigureMashupEnvironmentAndRun(Configuration configuration, Action action)
   at PQTest.Console.Program.RunTestBatteryCommand(CommandLineApplication app, CommandLineApplication command, Action`1 invoke)
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at PQTest.Console.Program.Main(String[] args)

````

### Desired Solution

Indication of which file contains the syntax error and (ideally) the line and column number of its location.

### Additional Context

The above example error was from a syntax problem in the .query.pq file.

**Comment by mattmasson** [comment]
This should be resolved in the [0.3.1 release](https://github.com/microsoft/vscode-powerquery-sdk/releases/tag/v0.3.1) (sdktools `2.127.2`).

Syntax errors should now be caught and displayed as an error.

The raw output looks like this:

```json
    {
        "ActivityId": "785b203d-6cf3-4ade-af76-f09e6d0d8422",
        "DataSourceAnalysis": [],
        "Details": "Token ',' expected. Start position: (3, 22). End position (3, 38).",
        "EndTime": "2024-03-18T22:05:19.6392845+00:00",
        "Method": "PQTest.RunTest",
        "Name": "test.query.pq",
        "StartTime": "2024-03-18T22:05:19.6228897+00:00",
        "DiagnosticEvents": {},
        "RowCount": 0,
        "Status": "Failed",
        "Type": "PQTest.Expression",
        "Error": {
            "Message": "Token ',' expected. Start position: (3, 22). End position (3, 38)."
        }
    }
```

**Comment by bgribaudo** [comment]
Thank you, @mattmasson! Unfortunately, the line numbers reported appear incorrect. Since I don't have permissions to reopen this issue, I'll open a new one.
